students/8/main.js

```diff
@@ -31,13 +31,33 @@ const ex5 = async () => {
 }
 
 const ex6 = async () => {
-    console.log("TODO...")
+    let result = await getPostWithName(11); // 1 is the post id.
+    console.log(result);
 }
 
 //
 // Your functions here...
 //
 
+// Ex 6
+
+const getPostWithName = async (Id) => {
+    let postName = null;
+    try {
+        let post = await axios.get(`https://jsonplaceholder.typicode.com/posts/${Id}`);
+        retval = post.data
+        user = post.data.userId
+        let userdata = await axios.get(`https://jsonplaceholder.typicode.com/users/${user}`);
```

Comment:
I think we can delete lines 48,49 and change line 50 to: 

` let userdata = await axios.get(`https://jsonplaceholder.typicode.com/users/${post.data.userId}`);`

and change 52 to:

`postName = Object.assign(post.data, retval2)`